docs/extenders/SecureCredentials.md

```diff
@@ -1,25 +1,15 @@
-# Secure Credentials API Adoption
+# Managing Zowe Explorer credential security
 
 Zowe Explorer extenders can adopt the Zowe Explorer Secure Credentials API to enable extensions to use Zowe profiles that are managed by the Secure Credential Store. The Secure Credentials API enables extenders to utilize the [Zowe security practices](https://docs.zowe.org/stable/user-guide/cli-scsplugin/) when performing z/OS actions outside of the core Zowe Explorer functions. The API is available in `@zowe/zowe-explorer-api` version 1.15.1 and higher.
 
-You can access the Zowe Explorer Secure Credentials API, using the `KeytarApi()` class, which handles profiles that are managed by the Secure Credential Store. Initialize the API before any extension is [registered with Zowe Explorer](<(../README-Extending.md#accessing-the-zowe-explorer-api)>), so that extenders can access the Data Sets, USS, and Jobs views.
+You can access the Zowe Explorer Secure Credentials API, using the `KeytarApi()` class. The class lets you handle profiles that are managed by Secure Credential Store. Initialize the API before any extension is [registered with Zowe Explorer](../README-Extending.md#accessing-the-zowe-explorer-extender-api), so that extenders can access the Data Sets, USS, and Jobs views.
 
-Initialize the `KeytarApi()`.
+## Prerequisites
 
-**Follow these steps:**
+Meet the following prerequisites:
 
-1. Ensure that the Zowe Explorer API is activated and initialized before you can use an extension. Define a VS Code extension dependency. For more information, see [Zowe Explorer extension dependencies and activation](../README-Extending.md#zowe-explorer-extension-dependencies-and-activation).
-2. Define an NPM dependency to the Zowe Explorer API in your VS Code extension `package.json` file to get access to Typescript type definitions provided for the API.
-3. Define an NPM dependency to Zowe CLI to get access to Typescript type definitions provided for Zowe Imperative.
-
-   ```json
-   "dependencies": {
-       "@zowe/cli": "6.30.0",
-       "@zowe/zowe-explorer-api": "1.16.0"
-   }
-   ```
-
-Get access to the initialized Zowe Explorer API objects that are provided by VS Code during or after activation. Use the following code snippet to gain access to `KeytarApi()`:
+- Ensure that the Zowe Explorer API is activated and initialized before you can use an extension. Define a VS Code extension dependency. For more information, see [Zowe Explorer extension dependencies and activation](../README-Extending.md#zowe-explorer-extension-dependencies-and-activation).
```

Comment:
This entire Pre-req section seems off to me, maybe we should just have the Prerequisites Header and point the user to the `Zowe Explorer extension dependencies and activation` docs. The following bulletpoint isn't a pre-req and should be like it was but possibly as # 4
![image](https://user-images.githubusercontent.com/49491949/160897771-c920ddd4-24be-472f-966b-cc44142eaa76.png)

Reply:
> Hey @JillieBeanSim, I'm confused by the second sentence of this paragraph. Should `Define a VSCE dependency` be its own requirement? (e.g. "Ensure that a VSCE dep is defined") OR Do we `Ensure that the ZE API is activated` **by** `"defining a VSCE dependency"` ?

No we do not "Ensure" that.  The VSC ext dependency is defined for extenders that require Zowe Explorer to be installed for it to work like CICS with adding a new View and FTP since all it does is add a new profile type to ZE.  For extensions that can work without ZE but may have added value when both extensions are installed may not have that requirement to auto install extensions but let their users know with the 2 installed together there is added value.

Reply:
after looking at the v2 conformance we have this as a requirement under "Extension Accessing Profiles" and wonder if this will need changing to best practice after my comment above about extensions that may just have added value when the 2 extensions are working together.  Thoughs @zFernand0 ?